src/blocks/content-helper/performance-details/component.tsx

```diff
@@ -96,9 +96,8 @@ function PerformanceDetailsSections( props: PerformanceSectionProps ) {
  * shown.
  */
 function DataPeriodSection(): JSX.Element {
-	const periodStart = getDateInUserLang(
+	const periodStart = getSmartShortDate(
```

Comment:
Using this for consistency across the PCH, so we show the year when the date isn't within the current year.